ipyvizzu.py

```diff
@@ -170,12 +174,10 @@ def animate(self, *animations: Animation, **options):
             raise ValueError("No animation was set.")
 
         animation = self._merge_animations(animations)
-        display_html(
-            self._ANIMATE.format(
-                id=id(self), animation=Animate(animation, options).dump()
-            ),
-            raw=True,
-        )
+        return self._show(Animate(animation, options).dump())
+
+    def restore(self, restore_id):
```

Comment:
Even the `restore` call will have a `restore_id`?

Reply:
Could you add a unit test?

Reply:
> Could you add a unit test?

Of course, it's just a draft right now.

Reply:
> Even the `restore` call will have a `restore_id`?

@nyirog 
`def restore(self, snapshot):` probably more clear

I work on it a bit because in this quick draft I 've created a new snapshot(snapshot_id) for a restore call too, but it is useless, because its param is the same snapshot
and also created for an animate.feature call, but it is not possible in this way (animate.feature is not a promise).

Reply:
I had a draft where `store` returns with a `Snapshot(Animation)`: https://github.com/vizzuhq/ipyvizzu/pull/47.

Is there a known issue with variable assignment inside the `chart.then( chart => {...} )` block?